caddyhttp/httpserver/plugin.go

```diff
@@ -328,6 +328,7 @@ var directives = []string{
 	"git",    // github.com/abiosoft/caddy-git
 
 	// directives that add middleware to the stack
+	"minify", // github.com/hacdias/caddy-minify
```

Comment:
Since minify can generate errors, it should at least go after the errors middleware. And you definitely don't want to be minifying after the gzip writer has closed.